docker-compose.yml

```diff
@@ -16,24 +16,19 @@ mysql:
     MYSQL_USER: teamcity
     MYSQL_PASSWORD: teamcity
     MYSQL_ALLOW_EMPTY_PASSWORD: yes
-agentone:
+agent:
   privileged: true
   image: alexanderilyin/docker-teamcity-agent
-  volumes:
-    - .TeamCity/.buildAgent/1/buildAgent.properties:/opt/buildAgent/conf/buildAgent.properties
-  links:
-   - server
-agenttwo:
-  privileged: true
-  image: alexanderilyin/docker-teamcity-agent
-  volumes:
-    - .TeamCity/.buildAgent/2/buildAgent.properties:/opt/buildAgent/conf/buildAgent.properties
+  command: -u 'http://server:8111/'
   links:
    - server
-agentthree:
-  privileged: true
-  image: alexanderilyin/docker-teamcity-agent
-  volumes:
-    - .TeamCity/.buildAgent/3/buildAgent.properties:/opt/buildAgent/conf/buildAgent.properties
+nginx:
+  image: quay.io/aptible/nginx
```

Comment:
@nafg why do we even need `nginx` here? I mean that it will allow you to access TeamCity on default `80` port but I'm not sure that it should be in this `*.yml`.